plugin-server/tests/main/process-event.test.ts

```diff
@@ -1711,7 +1711,7 @@ describe('when handling $create_alias', () => {
 
         // Make sure there is one non-identified person
```

Comment:
Example of why commenting tests can be problematic - this comment is now out of sync.